### ISSUE TITLE
Post-delete action is annoying

### ISSUE THREAD

**Issue by vinzv** [chimo/gs-staleAccounts#1]
After deleting a user on, let's say, page 16 one get's thrown back to page 1. That's a bit...inconvenient.

**Comment by chimo** [comment]
Agreed! Thanks for spotting this, I'll try to have a look at that tonight.

**Comment by chimo** [comment]
Quick update on this:
I think this might be a GS/delete-user behaviour that's not specific to this plugin.

I get the same effect if I go to page > 1 in the "User Directory" ( ex: https://gnusocial.de/directory/users/sort_by/nickname?page=2 ) and delete a user from there.

I'll have a closer look, but it might take a bit longer to fix if this is the case.
